sql_csv_utils.py

```diff
@@ -115,13 +115,16 @@ def get_one_hybrid(self, match, fullname):
             args:
                 match = the hybrid term of a taxonomic name e.g Genus A x B,
                         match - "A X B"
+                fullname = the full name of the taxonomic name.
         """
         parts = match.split()
         if len(parts) == 3:
+            basename = fullname.split()[0]
```

Comment:
now searches for basename as a part too, to avoid ambigous hybrid crosses with same names.